src/integrations/admin/import-integration.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+namespace Yoast\WP\SEO\Integrations\Admin;
+
+use WPSEO_Admin_Asset_Manager;
+use Yoast\WP\SEO\Conditionals\AIOSEO_V4_Importer_Conditional;
+use Yoast\WP\SEO\Conditionals\Yoast_Tools_Page_Conditional;
+use Yoast\WP\SEO\Conditionals\Import_Tool_Selected_Conditional;
+use Yoast\WP\SEO\Integrations\Integration_Interface;
+use Yoast\WP\SEO\Services\Importing\Importable_Detector;
+use Yoast\WP\SEO\Routes\Importing_Route;
+
+/**
+ * Loads import script when on the Tool's page.
+ */
+class Import_Integration implements Integration_Interface {
```

Comment:
I think we should add a few basic unit tests for this class to ensure code coverage is maintained.